### DISCUSSION TITLE
Arabic Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#334] [Language Leaders]
@Ahmed-farag36 are you still interested in being an Arabic language leader?

**Comment by Ahmed-farag36** [comment]
yes, I want to be the leader since I am the only contributor

**Comment by synesthesiam** [comment]
@Ahmed-farag36 adding you as language leader. Thanks!

**Comment by OmarHeshamACS** [comment]
Hello @synesthesiam  i want to volunteer in Arabic version also. I am a native Arabic Speaker and using Home Assistant widely in the past 3 years

**Reply by Ahmed-farag36** [reply]
Hello, Omar
Very good to help in supporting Arabic

**Comment by Natfaji** [comment]
Hi, i would like to be a language leader, please.

**Reply by synesthesiam** [reply]
Add you as a language leader. Welcome :slightly_smiling_face:
